file_walk_test.go

```diff
@@ -0,0 +1,43 @@
+package samosa
+
+import "testing"
+
+func Test_walk(t *testing.T) {
```

Comment:
Sorry for the nitpick, the test names should be in the form of mixed case.

Reference: https://go.dev/doc/effective_go#mixed-caps

Reply:
updated in commit https://github.com/burntcarrot/samosa/pull/4/commits/b7d53ff413ea3f4c3776c3f6e49c28fd0f5e3c5a